_includes/docs/extending/downloading.md

```diff
@@ -0,0 +1,25 @@
+Depending on, if you have a composer, command-line, drush-make or UI based workflow, downloading of themes and modules differs.
+
+### Composer
+Composer is a package manager for PHP and is the future standard for managing the dependencies of your Drupal 8 site and extensions. Go to the [Drupal documentation](https://www.drupal.org/docs/develop/using-composer/using-composer-with-drupal) for more information.
+
+If you have composer installed, go to the root of your site (here there should be a `composer.json` file) and install modules by typing `composer require drupal/[shortname of module]` into the command line.  
+For example: `composer require drupal/entity_browser`  
+You can also specify a version constraint here. Again refer to the [Drupal documentation about that topic](https://www.drupal.org/node/2718229#adding-modules) to learn more.
+
+### Command line
+To install extensions via the command line, without using composer, you need to have [drush](http://www.drush.org/en/master/) installed. With this tool, you can manage your site in different ways, e.g. importing/exporting sql dumps, updating translations, clearing the cache and also downloading and enabling modules and themes.
+
+The command to download an extension is `drush dl [extension]`, which will download the latest recommended release for your Drupal version. Execute `drush help dl` for all available options.
+
+### Drush make
+The drush-make workflow was the most used one, which involved not checking in all your dependencies into your repository,
```

Comment:
Don't understand this sentence. Plus, you should decide if it's "drush make" or "drush-make"